pytype/stubs/builtins/builtins.pytd

```diff
@@ -185,6 +185,7 @@ class object():
     __slots__ = []  # if used as base class
     __bases__ = ...  # type: Tuple[Any, ...]
     __dict__ = ...  # type: Dict[str, Any]
+    __dir__ = ... # type: List[str]
```

Comment:
This doesn't look quite right - `__dir__` is a method, so this should probably be `def __dir__(self) -> List[str]: ...`.